crates/ark/src/r_task.rs

```diff
@@ -253,7 +253,7 @@ where
     Fut: Future<Output = ()> + 'static,
 {
     // Idle tasks are always run from the read-console loop
-    if !only_idle && unsafe { R_TASK_BYPASS } {
+    if unsafe { R_TASK_BYPASS } {
```

Comment:
:P

I forgot I had to change that to make tests pass. In theory this only affects tests because of `R_TASK_BYPASS` and all tests seem to be passing normally.

If I don't add this, then the tests get deadlock as there's no R main thread to actually watch the task queue and execute them.